### PR TITLE
cc26

### PR DESCRIPTION
**Pull request by MarikoAlvarado**
- [x] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/insertionSort/insertionSort.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+insertionSort(arr){
+  for (let i = 1; i < arr.length; i++) {
+    let j = i - 1;
+    let temp = arr[i];
+    while (j >= 0 && temp < j) {
+
+    }
+  }
+
+}
```
Comment (adrianhuebner) [inline javascript/code-challenges/insertionSort/insertionSort.js:14]: This looks really great Mariko! I would love to see some tests written!
